app/models/tiny_mce_asset.rb

```diff
@@ -54,6 +54,14 @@ def url(style = :large, timeout: Constants::URL_LONG_EXPIRE_TIME)
     end
   end
 
+  def open
+    if image.is_stored_on_s3?
+      Kernel.open(presigned_url, 'rb')
+    else
+      File.open(file.path, 'rb')
```

Comment:
Again, I think that here the following should be written (otherwise it crashes for me!):
```ruby
File.open(image.path, 'rb')
```